pkg/cli/environment.go

```diff
@@ -77,6 +83,8 @@ func (s *EnvSettings) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVarP(&s.namespace, "namespace", "n", s.namespace, "namespace scope for this request")
 	fs.StringVar(&s.KubeConfig, "kubeconfig", "", "path to the kubeconfig file")
 	fs.StringVar(&s.KubeContext, "kube-context", s.KubeContext, "name of the kubeconfig context to use")
+	fs.StringVar(&s.Token, "token", s.Token, "bearer token used for authentication")
+	fs.StringVar(&s.APIServer, "server", s.APIServer, "the address and the port for the Kubernetes API server")
```

Comment:
Can we rename this to `kube-server` as per standard convention? e.g. `--kube-context`, `--kubeconfig`

This way we can differentiate between what comes from Helm, and what comes from Kubernetes.

Reply:
Valid point.
In that case we'll need to adapt the flag when passing it to kubectl during completion.  We'll need to adapt the following method:
https://github.com/helm/helm/blob/12e0ab7e44eb02738e1128ac01ee0843e47b4e79/cmd/helm/root.go#L62
@waveywaves This may be tricky, so let me know if you need help.

BTW, I'm working on making all this automatic, but it is not quite ready yet.

Reply:
@bacongobbler @marckhouzam Thanks for reviewing this. I will make the necessary changes.